### PR TITLE
TRAN-127: Added delivery hours funcionality, added translations, upda…

### PR DESCRIPTION
**Pull request by KarolinaKopacz**
**Add editable table for Delivery Hours**

**Implemented solution detail:**
TRAN-127

### CODE REVIEW

File: src/pages/menuUpload/customerMaster/CustomerMasterPage.tsx [hunk 1]
```diff
@@ -217,6 +229,14 @@ function CustomerMasterPage() {
     }
     payload.customer_type = formValues.customer_type.id;
     dispatch(putCustomerMasterRequest(payload as PutCustomerMasterRequestPayload));
+
+    if (customerDeliveryHours) {
+      const payloadDeliveryHours = { week_days: customerDeliveryHours, id: objectToEdit.id };
+      dispatch(
+        // @ts-ignore
```
Comment (sniedzielski) [inline src/pages/menuUpload/customerMaster/CustomerMasterPage.tsx:236]: @KarolinaKopacz Should we remove this?

Comment (KarolinaKopacz) [reply src/pages/menuUpload/customerMaster/CustomerMasterPage.tsx:236]: Not really, type-script despite passing on the type, doesn't see it so this is one way to pass the payload.

File: src/stores/sagas/customerWeekDaysSaga.ts [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+  postCustomerWeekDaysSuccess,
+} from 'stores/actions/customerWeekDays/customerWeekDaysActions';
+import CustomerWeekDaysActionTypes from 'stores/actions/customerWeekDays/customerWeekDaysTypes';
+import { sessionToken } from 'stores/reducers/tokenReducer';
+import refreshAccessToken from 'stores/sagas/utils';
+import apiClient, { getRequest } from 'utils/apiClient';
+
+export const customerWeekDaysUrl = 'customer_week_days/';
+
+export const postRequest = async (payload: object, id: number) => {
```
Comment (sniedzielski) [inline src/stores/sagas/customerWeekDaysSaga.ts:19]: @KarolinaKopacz in the future we could add this into 'apiClient.tsx'. But you don't have to do this now.

**Comment by sniedzielski** [comment]
@KarolinaKopacz We can merge this PR. There is no critical points that must be fixed ASAP. I left two comments, could be implemented later.

File: src/pages/menuUpload/customerMaster/CustomerMasterPage.tsx (hunk 1 above)
Comment (sniedzielski) [reply src/pages/menuUpload/customerMaster/CustomerMasterPage.tsx:236]: ok

**Review by sniedzielski — APPROVED** [review]

**Pull request merged**